src/sentry/hybridcloud/tasks/deliver_webhooks.py

```diff
@@ -134,7 +137,7 @@ def schedule_webhook_delivery() -> None:
         # Reschedule the records that we will attempt to deliver next.
         # We update schedule_for in an attempt to minimize races for potentially in-flight batches.
         mailbox_batch = (
-            WebhookPayload.objects.filter(id__gte=record["id"], mailbox_name=record["mailbox_name"])
+            WebhookPayloadReplica.filter(id__gte=record["id"], mailbox_name=record["mailbox_name"])
```

Comment:
note: there is an update on the loc below this

Reply:
That update should be fine. If records are available on the replica they can be mutated on the primary.